### DISCUSSION TITLE
when I installed torch2.0 and cu117,dreambooth training has gone wrong

### DISCUSSION THREAD

**Discussion by kakaxixx** [d8ahazard/sd_dreambooth_extension#647] [General]
when I upgraded Torch2.0+cu117,I found dreambooth can't train, No matter how many steps I trained, the new model is same as Original model, but if I use the old version, dreambooth is normal. Can you help me solve this problem? 

cu117 or torch2.0 has One does not support dreambooth?

**Comment by 1025307336** [comment]
Some people say that cude 11.7 and cude11.8 cannot implement a certain function of db, which will cause an error. The effective version is 11.3, you can try it

**Reply by hbqdev** [reply]
No, this seems to be a problem with torch2.0, i've been using torch 1.13.1 and that's been fine, only with torch 2.0 that I've also started having isues.
